src/pages/homepage/Home.jsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
```

Comment:
Suggestion: You can avoid writing import React on individual components as it is not necessary.